modules/sdk/gradle-plugins-defaults/CHANGELOG.markdown

```diff
@@ -2001,6 +2001,14 @@ plugin is no longer applied to OSGi projects.
 ### Changed
 - [LPS-72705]: Update the [Liferay Gradle Plugins] dependency to version 3.4.43.
 
+## 5.0.30 - 2017-08-29
+
+### Added
+- [LPS-73070]: Check and keep version in the `package-lock.json` file in sync.
```

Comment:
@Ithildir What version are we talking about here?

Reply:
The version of the module, synchronized with `Bundle-Version` in the `bnd.bnd`.